dev/cognitiveservices/data-plane/Language/analyzetext.json

```diff
@@ -1651,6 +1659,62 @@
       },
       "description": "Represents the confidence scores across all sentiment classes: positive and negative."
     },
+    "CustomSentimentAnalysisTaskParameters": {
+      "type": "object",
+      "description": "Supported parameters for a Custom Sentiment Analysis task.",
+      "properties": {
+        "stringIndexType": {
+          "$ref": "common.json#/definitions/StringIndexType"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/CustomTaskParameters"
+        }
+      ]
+    },
+    "CustomSentimentAnalysisLROTask": {
+      "type": "object",
+      "description": "Use custom models to ease the process of sentiment analysis from documents like product reviews",
+      "properties": {
+        "parameters": {
+          "$ref": "#/definitions/CustomSentimentAnalysisTaskParameters"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/AnalyzeTextLROTask"
+        }
+      ],
+      "x-ms-discriminator-value": "CustomSentimentAnalysis"
+    },
+    "CustomSentimentAnalysisResult": {
+      "type": "object",
+      "properties": {
+        "documents": {
+          "type": "array",
+          "description": "Sentiment analysis per document.",
+          "items": {
+            "allOf": [
+              {
+                "$ref": "#/definitions/SentimentDocumentResult"
```

Comment:
This object also describes the results for opinion mining and you said opinion mining is not part of this feature. This sharing could cause confusion. Could you please define your own result object that has things that are unique to this task only?

Reply:
Sounds good. Will do that!

Reply:
@jessicawrr any update on resolving this issue?

Reply:
@jessicawrr could you please address this comment?

Reply:
Oh Sorry. I missed this. I've updated the PR and defined the custom sa specific result